src/test/java/Homework18.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+        import org.openqa.selenium.WebElement;
+        import org.testng.Assert;
+        import org.testng.annotations.Test;
+
+
+public class Homework18 extends BaseTest{
+
+
+
+    @Test
+ public void playSong(){
+    navigateToPage();
+    provideEmail("natalia.kim01@testpro.io");
+    providePassword("AxKrdBnS");
+    clickSubmit();
+    clickPlay();
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

Reply:
Added.

Reply:
Please, re-submit homework  for review

Reply:
doing it right now, thank you

Reply:
re-submited.